turbinia/config/logger.py

```diff
@@ -38,6 +38,10 @@ def setup(need_file_handler=True, need_stream_handler=True):
   # Eliminate double logging from root logger
   logger.propagate = False
 
+  # Add turbinia logs to syslog
+  syslog_handler = logging.handlers.SysLogHandler(address = '/dev/log')
```

Comment:
this will only work on linux, insert a runtime check to see which platform we run on and adjust address accordingly?

Reply:
@rbdebeer Good point but is there any instance that we run Turbinia on non-linux environment?
Also, python's support for exporting logs to eventlogs isn't as good as Linux/Mac.

Reply:
Let's stick with this for now, in general most people will use the docker deployment. I'll put it on my list for Q1 to refactor as in general we should keep this as generic as possible, it's an external oss project.